src/components/workspace-utils.js

```diff
@@ -2,13 +2,15 @@ import debouncePromise from 'debounce-promise'
 import _ from 'lodash/fp'
 import { Fragment, useState } from 'react'
 import { b, div, h, p } from 'react-hyperscript-helpers'
-import { AsyncCreatableSelect, ButtonPrimary, ButtonSecondary, IdContainer, Link, Select, spinnerOverlay } from 'src/components/common'
+import { AsyncCreatableSelect, ButtonPrimary, ButtonSecondary, ClipboardButton, IdContainer, Link, Select, spinnerOverlay
+} from 'src/components/common'
```

Comment:
this linebreak causes a linting error, which is preventing tests from passing. https://github.com/DataBiosphere/terra-ui/wiki/Developing#code-style-and-linting has some info on configuring a local linter so you can see problems before PR - you may have to adapt those instructions for whatever editor you're using.

Reply:
I have no idea where that space came from but I think I've fixed it now.